application/src/main/java/org/thingsboard/server/service/subscription/TbSubscriptionsInfo.java

```diff
@@ -20,25 +20,24 @@
 import lombok.RequiredArgsConstructor;
 import lombok.ToString;
 
-import java.util.HashSet;
 import java.util.Set;
-import java.util.concurrent.ConcurrentHashMap;
 
 /**
  * Information about the local websocket subscriptions.
  */
 @RequiredArgsConstructor
 @AllArgsConstructor
-@EqualsAndHashCode(exclude = {"seqNumber"})
+@EqualsAndHashCode
```

Comment:
Remove all code that is not related to the scope of changes. Create new PR and cover this with TESTS!!!